pkg/services/accesscontrol/database/resource_permissions.go

```diff
@@ -11,81 +11,97 @@ import (
 	"github.com/grafana/grafana/pkg/services/sqlstore"
 )
 
-func (s *AccessControlStore) SetUserResourcePermissions(ctx context.Context, orgID, userID int64, cmd accesscontrol.SetResourcePermissionsCommand) ([]accesscontrol.ResourcePermission, error) {
+type flatResourcePermission struct {
+	ID          int64  `xorm:"id"`
+	ResourceID  string `xorm:"resource_id"`
+	RoleName    string
+	Action      string
+	Scope       string
+	UserId      int64
+	UserLogin   string
+	UserEmail   string
+	TeamId      int64
+	TeamEmail   string
+	Team        string
+	BuiltInRole string
+	Created     time.Time
+	Updated     time.Time
+}
+
+func (p *flatResourcePermission) Managed() bool {
```

Comment:
nit: I would prefer `IsManaged`, as I find prefixing functions that return bool with `is` easier to read.